src/playback/summary.jsx

```diff
@@ -169,13 +169,15 @@ class PlaybackSummary extends React.Component {
 
       const speed = result.speeds.find(s => s.loss > 1);
 
-      return `${speed.loss} dropped frames at ${speed.speed}x playback speed`;
+      return `${missing(speed.loss) ? speed.loss : speed.loss.toFixed(2)} 
```

Comment:
instead of 

    speed.loss.toFixed(2)

use 

    round(speed.loss, { places: 2 })

which will provide rounder numbers when `loss` is large.